src/NuGet.Clients/PackageManagement.UI/Xamls/ProjectView.xaml

```diff
@@ -78,7 +78,7 @@
       MinHeight="24"
       AutomationProperties.AutomationId="Project_Button_Uninstall"
       HorizontalAlignment="Left"
-      Visibility="{Binding InstalledVersion, Converter={StaticResource NullToVisibilityConverter}, Mode=OneWay}"
```

Comment:
yes, so i wouldn't remove the Visibility property from here. For projects that are AutoReferenced, the InstalledVersion won't be null, right?

---
In reply to: [98809702](https://github.com/NuGet/NuGet.Client/pull/1158#discussion_r98809702) [](ancestors = 98809702)